client/src/pages/Imprint.js

```diff
@@ -0,0 +1,31 @@
+import "./Imprint.css";
+import Card from "../components/Card";
+import BackButton from "../components/BackButton";
+
+export default function Imprint({ isLight }) {
+  return (
+    <main id="Imprint" className="card-screen">
```

Comment:
you shouldn't use the `id` here for styling purposes. Also, it'd be nice if you always follow the same convention: 
`className` of the root element of a component = the component name